app/views/bars/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>Add Bar</h1>
+    <%= form_for @bar do |f| %>
```

Comment:
This line should not be indented! The `erb` tags should be in line with the `br` tags